### ISSUE TITLE
Ability to bulk create trusted publishers

### ISSUE THREAD

**Issue by ekohl** [rubygems/rubygems.org#4309]
## Is your feature request related to a problem?

As an admin of both [Vox Pupuli](https://rubygems.org/profiles/voxpupuli) and [The Foreman](https://rubygems.org/profiles/theforeman) I'm responsible for a sizeable number of gems. Clicking through the UI is tedious and error prone, so I'd like some programmatic way to set up in bulk.

## Describe the solution you'd like

A way to both set up trusted publishing and verify it is indeed set up in a programmatic way.

Perhaps via the `gem` application, similar to how `gem owner` allows managing the owner. That allows me to use shell scripts in a loop. Only a REST API would be a good start.

**Comment by simi** [comment]
@segiddins any plans on trusted publishers API?

**Comment by segiddins** [comment]
Yup, we can set up an API. It'll probably necessitate adding a new API scope, and also ensuring MFA is given.

**Comment by segiddins** [comment]
Done in https://github.com/rubygems/rubygems.org/pull/4690

**Comment by ekohl** [comment]
Thanks for implementing this. Is there any documentation on how to use this?

**Comment by segiddins** [comment]
I don't believe so, but it appears https://guides.rubygems.org/rubygems-org-api/ would be a good place to add an entry!
